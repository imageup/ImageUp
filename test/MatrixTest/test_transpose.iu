function main() -> int
{
    matrix: mat(3,4) = [1,2,3,4|5,6,7,8|9,10,11,12];
    printf(mat[0][0]);
    printf(mat[0][1]);
    printf(mat[0][2]);
    printf(mat[0][3]);
    transpose(mat);
    printf(mat[0][0]);
    printf(mat[1][0]);
    printf(mat[2][0]);
    printf(mat[3][0]);
    return 0;
}