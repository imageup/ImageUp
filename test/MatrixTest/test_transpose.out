llc -relocation-model=pic test.ll > test.s ;gcc -o test.exe test.s lib.c `pkg-config --cflags --libs opencv`;./test.exe
1
2
3
4
1
2
3
4
